src/ui/screens/settings/ViewSettingsMenu.qml

```diff
@@ -136,7 +151,7 @@ MZViewBase {
                 imageRightSrc: "qrc:/nebula/resources/chevron.svg"
                 imageRightMirror: MZLocalizer.isRightToLeft
                 onClicked: {
-                    Glean.sample.settingsAboutUsOpened.record();
```

Comment:
The metrics that we are removing should be removed from the `metrics.yaml` as well.